### PR TITLE
[[ ServerCommunityPlus ]] Create additional zip file for communityplus server engine

### PR DESCRIPTION
**Pull request by livecodeian**

### CODE REVIEW

File: builder/server_builder.livecodescript [hunk 1]
```diff
@@ -110,20 +110,12 @@ command serverBuilderRun pPlatform, pEdition
...
          put "livecode-community-server" & tOutExeExtension into tServerZipPath
-         break
-      case "Indy"
-         put tEngineFolder & slash & "server-commercial" & tExeExtension into tServerPath
-         put "livecode-server" & tOutExeExtension into tServerZipPath
-         break
-      case "Business"
-         put tEngineFolder & slash & "server-professional" & tExeExtension into tServerPath
+      else
          put "livecode-server" & tOutExeExtension into tServerZipPath
```
Comment (montegoulding) [inline builder/server_builder.livecodescript:117]: @livecodeian am I missing something or does this mean the commercial server edition zips will clobber each other?

Comment (montegoulding) [reply builder/server_builder.livecodescript:117]: Oh I see the file name in the archive will be the same... hmm... if there's a reason to make community different here perhaps they should all be different?

Comment (livecodeian) [reply builder/server_builder.livecodescript:117]: Perhaps - though the naming differences between community / commercial was there before this patch and I usually err on the side of being conservative when making these kinds of changes.

Comment (montegoulding) [reply builder/server_builder.livecodescript:117]: ok, perhaps we should discuss internally as it's either important to indicate the edition and therefore license in the file name or it isn't so it seems like they should all be the same or all different.

**Comment by montegoulding** [comment]
@livecode-vulcan review ok 71a2335

**Comment by livecode-vulcan** [comment]
💙 review by @montegoulding ok 71a2335628266eb99d90f77efec4146c13de996e

**Comment by livecode-vulcan** [comment]
😎 test success 71a2335628266eb99d90f77efec4146c13de996e

* try-community-armv6-android-api8: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api8/builds/989)
* try-community-armv6-android-api9: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api9/builds/1005)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/979)
* try-community-universal-ios-iphoneos11.1: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos11.1/builds/52)
* try-community-universal-ios-iphonesimulator11.1: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator11.1/builds/52)
* try-community-universal-mac-macosx10.6: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.6/builds/984)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/1006)
* try-community-x86-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian7/builds/989)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/1005)
* try-community-x86_64-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian7/builds/988)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/998)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/1021)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/985)

**Pull request merged**